integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/receipt/ConversationService.java

```diff
@@ -53,6 +53,7 @@ public ConversationService(ConversationRepository repo,
     }
 
     public Conversation registerStatus(Conversation conversation, MessageStatus status) {
+        System.out.println("AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS106)